fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -314,7 +314,7 @@
     private boolean enableJoinReorderBasedCost = false;
 
     @VariableMgr.VarAttr(name = FORWARD_TO_MASTER)
-    public boolean forwardToMaster = false;
+    public boolean forwardToMaster = true;
```

Comment:
default is true?? The query will also forward to master???

Reply:
Only effect the following stmts
![image](https://user-images.githubusercontent.com/9098473/140468227-fbc59fc7-dd9f-4f25-ba6e-e914d8545911.png)